pkg/oc/admin/policy/cani.go

```diff
@@ -71,6 +71,7 @@ func NewCmdCanI(name, fullName string, f *clientcmd.Factory, out io.Writer) *cob
 				os.Exit(2)
 			}
 		},
+		Deprecated: fmt.Sprintf("Use oc auth can-i"),
```

Comment:
@simo5 Mind adding single quotes around the new `oc` command?

Reply:
Can you also use the `moved` function where these commands are specifed, similarly to [this](https://github.com/openshift/origin/pull/18093/files#diff-98f0c62da95151f788ac0edc23423802R200). This way when you invoke `oc policy` these deprecated commands will be appropriately marked as well.

Reply:
> @simo5 Mind adding single quotes around the new oc command?

@juanvallejo why? I don't recall seeing oc in quotes anywhere

Reply:
One more thing, use lowercase, since this gets appended to the following sentence `Command xyz is deprecated, <here goes your string>`

Reply:
@soltysh 
> why? I don't recall seeing oc in quotes anywhere

Saw this being the pattern in a few places:
- https://github.com/openshift/origin/blob/master/pkg/oc/admin/admin.go#L134
- https://github.com/openshift/origin/blob/master/pkg/oc/admin/node/node.go#L98
- https://github.com/openshift/origin/blob/master/pkg/oc/cli/cli.go#L281

Reply:
I looked into cobra (see [here](https://github.com/openshift/origin/blob/5b71e8d8e441a0d39f9228a4bce4c7a6a13be08b/vendor/github.com/spf13/cobra/command.go#L546-L548)) and that's why it does make sense to be it lowercase.